### PR TITLE
Cleanup importor and migrate the java client

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -397,12 +395,6 @@ function(nebula_add_subdirectory dir_name)
     add_subdirectory(${dir_name})
 endfunction()
 
-if (NOT SKIP_JAVA_CLIENT)
```
Comment (laura-ding) [inline CMakeLists.txt]: also need `set(NEBULA_CLEAN_ALL_DEPS clean-interface clean-pch clean-hbase)`

Comment (darionyaphet) [reply CMakeLists.txt]: thanks & fixed

**Comment by jude-zhu** [comment]
Jenkins go

**Comment by monadbobo** [comment]
LGTM.

**Pull request merged**